k8s/cspc_install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "Running pre-installation steps"
+
+x=$(kubectl get crd cstorpoolclusters.openebs.io -o jsonpath="{.spec.scope}" 2>&1 )
+
+if [[ $x == 'Cluster' ]]; then
+    kubectl delete crd cstorpoolclusters.openebs.io
+fi
+
+echo "Installing cspc operator"
+
+kubectl apply -f cspc-operator.yaml
```

Comment:
better to change it to full raw github link

Reply:
again, it depends on our doc steps..
if our doc says to download and run, then, this should be fine

Reply:
added the raw github link instead as there is nothing that user can customize in this yaml